app/controllers/users/registrations_controller.rb

```diff
@@ -42,7 +42,7 @@ class Users::RegistrationsController < Devise::RegistrationsController
 
   # If you have extra params to permit, append them to the sanitizer.
   # def configure_sign_up_params
-  #   devise_parameter_sanitizer.permit(:sign_up, keys: [:attribute])
+  #   devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name])
```

Comment:
why did you change this code? hehe

Reply:
My bad, maam. This was one of my attempts with addressing the unpermitted parameters and I forgot to change this back